lib/mandrill_dm/message.rb

```diff
@@ -248,6 +258,10 @@ def return_string_value(field)
       mail[field] ? mail[field].to_s : nil
     end
 
+    def return_time_parsed_string_value(field)
+      mail[field] ? Time.parse(mail[field].value).utc.strftime("%Y-%m-%d %H:%M:%S") : nil
```

Comment:
value will never be a `Time` object - it's cast by the Mail gem to a string (as is Date and DateTime). This parses back to a Time object so that we can format it as Mandrill expects.